dozer/leak.py

```diff
@@ -192,7 +198,9 @@ def index(self, req):
                        )
                 rows.append(row)
         res = Response()
-        res.text = template(req, "graphs.html", output="\n".join(rows))
+        res.text = template(
+            req, "graphs.html", output="\n".join(rows), floor=floor, filter=filtertext
```

Comment:
I wonder if `filtertext` needs to be passed to `escape()` here.

We know that `floor` is safe because `int(floor)` did not raise earlier.

Reply:
Right, fixed in 90b6bf7.